linkis-dist/package/db/linkis_ddl.sql

```diff
@@ -907,6 +911,9 @@ CREATE TABLE `linkis_ps_dm_datasource_type`
     `classifier`  varchar(32) COLLATE utf8_bin NOT NULL,
     `icon`        varchar(255) COLLATE utf8_bin DEFAULT NULL,
     `layers`      int(3)                       NOT NULL,
+    `description_en` varchar(255) DEFAULT NULL COMMENT 'english description',
```

Comment:
do we need to add alter command of `linkis_ps_dm_datasource_type ` in our upgrade sql file?

Reply:
Yes, this is already in package/db/upgrade/1.3.2_schema/mysql/linkis_ddl.sql